src/cnc/models/config/resource.py

```diff
@@ -505,6 +509,23 @@ def queue_name(self):
 
         return queue_name
 
+class DynamoDBResourceSettings(BaseResourceSettings):
+    type: Literal["dynamodb"]
+    billing_mode: Optional[Literal["PROVISIONED", "PAY_PER_REQUEST"]] = "PAY_PER_REQUEST"
+    hash_key : Optional[str] = None
+    read_capacity : Optional[int] = 5
+    write_capacity : Optional[int] = 5
+    table_class : Optional[Literal["STANDARD", "STANDARD_INFREQUENT_ACCESS"]] = "STANDARD"
+    deletion_protection_enabled : Optional[str] = None
+
+    @property
+    def is_dynamodb(self):
+        return True
+
+    @property
```

Comment:
Should export its ARN I think here or whatever else is used to connect to the table

Reply:
Modifications Reference: https://github.com/coherenceplatform/cnc/pull/90/commits/7e62798906dcb1450236260512a3de5636f93152